cpp/src/Ice/Object.cpp

```diff
@@ -49,94 +49,99 @@ Ice::Object::ice_staticId()
     return typeId;
 }
 
-bool
-Ice::Object::_iceD_ice_isA(Incoming& incoming)
+void
+Ice::Object::_iceD_ice_isA(IncomingRequest& request, function<void(OutgoingResponse)> sendResponse)
 {
-    InputStream* istr = incoming.startReadParams();
+    InputStream* istr = &request.inputStream();
```

Comment:
Can't we use a the reference instead of getting a pointer here?

Reply:
We don't because all the generated code for decoding (and encoding) is pointer based. Same for all the StreamHelpers. This code in Object.cpp is / should be like the generated code.